### DISCUSSION TITLE
How does this note hide the rhythmic notation

### DISCUSSION THREAD

**Discussion by ifccod** [CoderLine/alphaTab#1139] [Q&A]
dear author：
if  rhythmMode=ShowWithBeams
How does this note hide the rhythmic notation

I want to render it in tex syntax

![image](https://user-images.githubusercontent.com/102718410/229773395-5ab5df11-0a24-431f-9368-324c8b6b071c.png)
![image](https://user-images.githubusercontent.com/102718410/229773454-6ce16721-3edf-4e80-a0db-e4dbc4325a9d.png)

**Comment by Danielku15** [comment]
From how it looks, I think it might be a grace note: https://alphatab.net/docs/alphatex/beat-effects#simple-effects 
For those kind of notes, no rhythm notation is shown. But to be sure: Can you provide the Guitar Pro file you used in the screenshot?
